### PR TITLE
[ja] translation orm/deleting-data.rst

### PR DESCRIPTION
**Pull request by boarnasia**
Hi, there.

I've translated orm/deleting-data.rst.

### CODE REVIEW

File: ja/orm/deleting-data.rst [hunk 1]
```diff
@@ -8,13 +8,56 @@ Deleting Data
...
-    button to directly propose your changes.
+    ``cascadeCallbacks`` が ``true`` の時、一括削除に比較して削除処理はだいぶ遅くなります。cascadeCallbacks オプションは、あなたのアプリケーションで event listerner によるエベント処理が重要な働きを持つ場合のみ有効にされるべきです。
+
+
+一括削除
+-----------------
+
+.. php:method:: deleteAll($conditions)
+
+一行づつ削除することが効率的でなかったり有用ではない時があります。そういったケースでは、一回で複数行を削除するために、bulk-delete を使うことが効率的です::
```
Comment (okinaka) [inline ja/orm/deleting-data.rst]: "bulk-delete" means "一括削除".

File: ja/orm/deleting-data.rst [hunk 2]
```diff
@@ -8,13 +8,56 @@ Deleting Data
...
+
+    // 全てのスパムを削除する
+    function destroySpam()
+    {
+        return $this->deleteAll(['is_spam' => true]);
+    }
+
+bulk-delete では、１つ以上の行が削除されると成功したとみなされます。
```
Comment (okinaka) [inline ja/orm/deleting-data.rst]: "bulk-delete" means "一括削除".

File: ja/orm/deleting-data.rst [hunk 3]
```diff
@@ -8,13 +8,56 @@ Deleting Data
...
+6. ``Model.afterDelete`` イベントが起動されます。
+
+デフォルトでは、一回のトランザクションの中で全ての削除が行われますが、atomic オプションで無効化することも出来ます::
+
+    $result = $this->Articles->delete($entity, ['atomic' => false]);
+
+連鎖削除
+-----------------
+
+エンティティを削除するとき関連データも同時に削除されます。HasOne、HasMany が ``dependent`` として設定されている場合、削除処理はそれらのエンティティにも連鎖適応されます。デフォルトでは、関連テーブル内のエンティティは :php:meth:`Cake\\ORM\\Table::deleteAll()` を使うことで削除されます。その際に、関連するエンティティを逐次読み込み、削除イベントを個別に起動させるには ``cascadecallbacks`` オプションを ``true`` にセットします。上記２つのオプションを有効にした HasMany のサンプルは、このようになります::
```
Comment (boarnasia) [inline ja/orm/deleting-data.rst]: thank you for checking.
I'll revise those soon.

File: ja/orm/deleting-data.rst [hunk 4]
```diff
@@ -8,13 +8,56 @@ Deleting Data
...
+    ]);
+
 .. note::
-    The documentation is not currently supported in Japanese language for this
-    page.
 
-    Please feel free to send us a pull request on
-    `Github <https://github.com/cakephp/docs>`_ or use the **Improve This Doc**
-    button to directly propose your changes.
+    ``cascadeCallbacks`` が ``true`` の時、一括削除に比較して削除処理はだいぶ遅くなります。cascadeCallbacks オプションは、あなたのアプリケーションで event listerner によるエベント処理が重要な働きを持つ場合のみ有効にされるべきです。
```
Comment (chinpei215) [inline ja/orm/deleting-data.rst]: > event listerner によるエベント処理

Typo. And, if I were you, I would translate it into something like this:

> when your application has important work handled by event listeners.

あなたのアプリケーションがイベントリスナーによって処理される重要な仕事を持っている場合

as `event` is not repeated in the original sentence.

**Comment by chinpei215** [comment]
@boarnasia Thank you for your contribution. Please check out my review.
By the way, why several unrelated commits are contained in this pull request? I am not sure but you might want to rebase your commits.

**Comment by boarnasia** [comment]
@chinpei215 Thanks for reviewing my commits.

> By the way, why are several unrelated commits contained in this pull request? I am not sure but you might want to rebase your commits.
Your guess is right.
I just wanted to rebase, but other's commits ware included.
I'm sorry for confusing you, I'm not familiar with "Pull request" yet.

**Comment by okinaka** [comment]
You can remove the unrelated commits using "git rebase -i" in your branch.
And add "-f" option when "git push" into your github repository.

File: ja/orm/deleting-data.rst [hunk 5]
```diff
@@ -8,13 +8,56 @@ Deleting Data
...
+    $result = $this->Articles->delete($entity);
+
+エンティティ削除時には、いくつかのことが起こります:
+
+1. :ref:`削除ルール <application-rules>` が適用されます。 ルールのチェックに失敗した場合、削除は中止されます。
+2. ``Model.beforeDelete`` イベントが起動されます。このイベントが停止した場合、削除は失敗し、イベントの戻り値が返されます。
+3. エンティティが削除されます。
+4. 全ての依存関係先が削除されます。依存関係先がエンティティとして削除されるとき、追加のイベントが起動されます。
+5. BelongsToMany アソシエーション用の全ての結合テーブルのレコードが削除されます。
+5. 全ての結合テーブルのレコードが削除されます。
```
Comment (chinpei215) [inline ja/orm/deleting-data.rst]: No 5. is duplicated.

**Comment by boarnasia** [comment]
@okinaka Thank you for your advice.
I'll try it.

File: ja/orm/deleting-data.rst (hunk 5 above)
Comment (boarnasia) [inline ja/orm/deleting-data.rst]: I've fixed it.

**Comment by okinaka** [comment]
@boarnasia Thanks!

**Comment by boarnasia** [comment]
@okinaka @chinpei215  It's my pleasure.

**Pull request merged**